### ISSUE TITLE
FileNotFoundError: [Errno 2] 

### ISSUE THREAD

**Issue by gerroon** [carson-katri/dream-textures#539]
### Description

It looks like there is a slash issue with the paths on Windows.  I have the folder under 

```C:\\Users\\USER\.cache\\huggingface\\diffusers\\models--stabilityai--stable-diffusion-x4-upscaler\\refs\\main``

Actual file

`C:\Users\USER\.cache\huggingface\diffusers\models--stabilityai--stable-diffusion-x4-upscaler\refs\fp16`

The code seems to generate with reverse slash before .cache

```C:\\Users\\USER/.cache\\huggingface\\diffusers\\models--stabilityai--stable-diffusion-x4-upscaler\\refs\\main```

I am Windows 11 Blender 4.5,  DT 0.0.9

```
The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream_textures\generator_process\actor.py", line 111, in run_callbacks
    done_callback(self)
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream_textures\preferences.py", line 185, in on_done
    set_model_list('installed_models', future.result())
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream_textures\generator_process\actor.py", line 58, in result
    raise self._exception
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\USER/.cache\\huggingface\\diffusers\\models--stabilityai--stable-diffusion-x4-upscaler\\refs\\main'

```

### Steps to Reproduce

Enable the addon

Start Blender from the Console

### Expected Behavior

It should find the correct files

### Addon Version

Windows (CUDA)

### GPU

NVIDIA

**Comment by JasonHoku** [comment]
That sure is a weird one. Maybe a Win11 thing, maybe some mismatched dependency versioning. Did you dl the prebuilt version?

You could try modifying `/generator_process/actions/huggingface_hub.py` where at line 67 you could change the DIFFUSERS_CACHE variable to point to the actual path.

**Comment by Shreyash113** [comment]
hey same error any resolves im on windows 10 though

**Comment by gerroon** [comment]
Those seem to work well for my local Stable Diffusion server. I was hoping to not duplicate data on my server.

**Comment by carson-katri** [comment]
This was resolved in #479 and will be included in the next release
